docs/content/en/docs/pipeline-stages/builders/buildpacks.md

```diff
@@ -63,4 +59,12 @@ buildpack:
     - src/*.go
     ignore:
     - vendor/**
-```
\ No newline at end of file
+```
+
+### Limitations
+
+`skaffold debug` is unable to configure the container images produced
+by Cloud Native Buildpacks for debugging.  Images produced by the
+produced by Cloud Native Buildpacks use a `launcher` binary that
```

Comment:
```suggestion
 Cloud Native Buildpacks use a `launcher` binary that
```

Reply:
oh jeepers.  I honestly did reread this, several times.